examples/js/loaders/GLTFLoader.js

```diff
@@ -533,6 +538,55 @@ THREE.GLTFLoader = ( function () {
 
 	};
 
+	/**
+	 * Texture Transform Extension
+	 *
+	 * Specification:
+	 */
+	function GLTFTextureTransformExtension( json ) {
+
+		this.name = EXTENSIONS.KHR_TEXTURE_TRANSFORM;
+
+	}
+
+	GLTFTextureTransformExtension.prototype.extendTexture = function ( texture, mapDef ) {
```

Comment:
nit: How about just taking `extensions` as an parameter, rather than `mapDef`? Seems like only `.extensions` of `mapDef` is used in this function.

Reply:
Done ✅